gax/BUILD.bazel

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Google Inc.  All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+cc_library(
+    name = "cpp_retry_policy",
+    srcs = [],
+    hdrs = [
+        "retry_policy.h",
+    ],
+    deps = [
+    ],
+)
+
+cc_test(
```

Comment:
Can you follow similar syntax to the generator build file to create a list of tests? (https://github.com/googleapis/gapic-generator-cpp/blob/master/generator/BUILD.bazel#L62) Although this list is only 1 entry long now, this will let us add tests easily and avoid polluting the build file too much.

Reply:
Done.